CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif()
 
 find_package(Eigen3 REQUIRED)
 find_package(TBB REQUIRED)
-find_package(pybind11 CONFIG REQUIRED)
```

Comment:
Use find_package here. We will use Python build systems to install pybind11 or trust that a user has installed it.